aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -59,6 +69,27 @@ void convert_indices_from_coo_to_csr_cpu(const Tensor& result, const Tensor& inp
     data_out[i] = static_cast<output_t>(numel);
 }
 
+template <typename input_t, typename output_t>
+void convert_indices_from_csr_to_coo_cpu(const Tensor& result, const Tensor& crow_indices, const Tensor& col_indices, const int64_t size) {
+  int64_t numel = crow_indices.numel();
+  const input_t* col_indices_data_in = col_indices.data_ptr<input_t>();
```

Comment:
For this to work these need to be contiguous. I'd either check or just simply cast via contiguous.

Reply:
Oh yeah, that makes sense. [Added contiguous calls](https://github.com/pytorch/pytorch/pull/66774/commits/259062eebc86fdb90ee3d1bafa88e854d22c113d).